packages/cli/src/lib/access/access-utils.ts

```diff
@@ -0,0 +1,11 @@
+export const isTeamApp = function (owner: string | undefined) {
```

Comment:
Since this already exists in `/lib/access` should we just call this file `utils.ts`?

Reply:
I went back and forth on that, but I'm ok making this change.